scripting/cwx.sp

```diff
@@ -632,6 +633,7 @@ int Native_RemovePlayerLoadoutItem(Handle plugin, int argc) {
 	int flags = GetNativeCell(4);
 	
 	UnsetClientCustomLoadoutItem(client, playerClass, itemSlot, flags);
+	return true;
```

Comment:
`CWX_RemovePlayerLoadoutItem` returns void, so if anything this should return 0.